Server/Controllers/epicController.js

```diff
@@ -62,6 +62,14 @@ class epicMail {
       data: array,
     });
   }
+
+  static getallsentmessages(req, res) {
```

Comment:
this method name should be in camel case

Reply:
I will change it as soon as possible, thanks